wrapper/src/main/java/software/amazon/jdbc/plugin/strategy/fastestresponse/NodeResponseTimeMonitor.java

```diff
@@ -122,7 +128,7 @@ public void monitor() {
               break;
             }
             long startTime = this.getCurrentTime();
-            if (this.pluginService.getTargetDriverDialect().ping(this.monitoringConn)) {
+            if (this.pluginService.getTargetDriverDialect().ping(this.monitoringConn.get())) {
```

Comment:
same as above, do we need a local variable here in case monitoringConn becomes null between the check above and the usage here?